src/main/java/org/ideoholic/curium/model/appointment/service/AppointmentService.java

```diff
@@ -117,34 +114,25 @@ public ResultResponse addAppointment(AddAppointmentDto addAppointmentDto) {
 		return ResultResponse.builder().build();
 	}
 
-	
-	public boolean viewAllAppointments() {
-
-		boolean result = false;
+	public ViewAllAppoinmentsResponseDto viewAllAppointments(ViewAllAppointmentsDto viewAllAppointmentsDto) {
+           ViewAllAppoinmentsResponseDto viewAllAppoinmentsResponseDto = new ViewAllAppoinmentsResponseDto();
 		//String pages = "1";
-		if(httpSession.getAttribute("branchid")!=null){
+		if(viewAllAppointmentsDto.getBranchId()!=null){
 			try {
 				int page = 1;
 				int recordsPerPage = 500;
-					if (!"".equalsIgnoreCase(DataUtil.emptyString(request.getParameter("page")))) {
-						page = Integer.parseInt(request.getParameter("page"));
+					if (!"".equalsIgnoreCase(DataUtil.emptyString(String.valueOf(viewAllAppointmentsDto.getPage())))) {
+						page = viewAllAppointmentsDto.getPage();
 					}
-
 				List<Appointment> list = new AppointmentDAO().readListOfObjectsPagination((page - 1) * recordsPerPage,
-						recordsPerPage, Integer.parseInt(httpSession.getAttribute("branchid").toString()));
-				request.setAttribute("studentList", list);
```

Comment:
This `request.setAttribute("studentList", list);` should be set as `viewAllAppoinmentsResponseDto.setStudentList(list)` object.